dbms/src/TiDB/Schema/TiDBSchemaSyncer.h

```diff
@@ -213,6 +213,14 @@ struct TiDBSchemaSyncer : public SchemaSyncer
                     LOG_WARNING(log, "Skip the schema diff from version {}. ", cur_version + diff_index + 1);
                     continue;
                 }
+
+                if (schema_diff->regenerate_schema_map)
+                {
+                    // If `schema_diff.regenerate_schema_map` == true, return `-1` direclty, let TiFlash reload schema info from TiKV.
```

Comment:
Please also add related comments in L163 and L126, to make the code more easily to understand.

Reply:
L163 updated, L126 is not related with `flashback`.